src/network/mcpe/handler/PreSpawnPacketHandler.php

```diff
@@ -98,7 +98,7 @@ public function setUp() : void{
 				$this->server->getMotd(),
 				"",
 				false,
-				new PlayerMovementSettings(PlayerMovementType::SERVER_AUTHORITATIVE_V1, 0, false),
+				new PlayerMovementSettings(ServerAuthMovementMode::SERVER_AUTHORITATIVE_V2, 0, false),
```

Comment:
Why is this changed to v2?

Reply:
It is correct, the old [one](https://github.com/pmmp/BedrockProtocol/blob/9825e3abdd43f13f54aa359cce5020c5e303541c/src/types/PlayerMovementType.php#L20)'s value was `1` and so is the [new](https://github.com/pmmp/BedrockProtocol/blob/6aa7cbeb4a7ec6fa58f9024aeaddad7c5c65a459/src/types/ServerAuthMovementMode.php#L21)

Reply:
Thanks